D1371R3.md

```diff
@@ -371,7 +371,7 @@ Op parseOp(Parser& parser) {
 ## Basic Syntax
 
 > | `inspect constexpr`*~opt~* `(` *init-statement~opt~* *condition* `)` *trailing-return-type~opt~* `{`
-> |     *pattern* *guard~opt~* `=>` *statement*
+> |     *pattern* *guard~opt~* `=>` *statement* `;`
```

Comment:
`statement` here is correct. if it happens to be an `expresssion statement`, the expression statement is what contains the `;`. if it happens to be a compound-statement, we don't want it to be `{ ... };`.

Reply:
Thanks Michael. I did wonder about this. Will fix and merge.